### PR TITLE
[Live-Site]: Onboarding flow for the users

### LINKED ISSUE #536: [Live-Site] Move to room codes
## Description
Currently the roles available on the live site are host and guest. And 
The URL to join the host is:
```bash
/live?role=host
```
To join the guest the URL is:
 ```bash
/live?role=guest&room=<room-id>
```
The problem with this approach is that it gives the user many implementation details about the role and the room id.

## Requirement
- Add two more roles MAVEN, MODERATOR
- Use the room code to join the user. So, the URL will be `/live/<room_code>`
- Host, Maven, and Moderator will have options to copy the invite link

## User flow
user-flow-2022-03-05-1253

## UX
https://www.figma.com/file/2VtPGJH61Qp0iarkKW6KTO/Live-Site-Detailed-Designs?type=design&node-id=0-1&mode=design&t=gbAlOHD0tf6FHHvt-0

## Reference
https://www.100ms.live/docs/server-side/v2/api-reference/room-codes/room-code-overview

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### What is the change?
- Select a role modal for users
- Added room codes for Maven
- Create a room code modal for the host

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

https://github.com/Real-Dev-Squad/website-www/assets/40385118/6091e6fc-250e-4b66-8d00-82e9476bbca8

### CODE REVIEW

**Review by satyam73 — DISMISSED** [review]
LGTM

**Review by Pratiyushkumar — DISMISSED** [review]

File: app/constants/urls.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ const APP_URLS = {
     MEMBERS: `${SCHEME}staging-members.${DOMAIN}`,
     STATUS: `${SCHEME}staging-status.${DOMAIN}`,
     PROFILE: `${SCHEME}staging-my.${DOMAIN}`,
-    API_BACKEND: `${SCHEME}staging-api.${DOMAIN}`,
+    // API_BACKEND: `${SCHEME}staging-api.${DOMAIN}`,
+    API_BACKEND: `http://localhost:3000`,
```
Comment (rohan09-raj) [inline app/constants/urls.js:33]: Can we please revert this change ?

**Review by rohan09-raj — APPROVED** [review]
